Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -234,18 +271,52 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
         },
 
         "'platform' zone": function () {
-            var value = this.service.get('platformNavigationItems');
+            var value;
+
+            this.rootStruct = {'location': new Mock(), 'content': new Mock()};
+
+            Mock.expect(this.rootStruct.location, {
+                method: 'get',
+                args: ['id'],
+                returns: '/allez/om',
+            });
+
+            Mock.expect(this.rootStruct.content, {
+                method: 'get',
+                args: [Mock.Value.Any]
```

Comment:
same as https://github.com/ezsystems/PlatformUIBundle/pull/233/files#r34234146 and in addition, here the mock should return a value a